### ISSUE TITLE
Error parsing payload in jee

### ISSUE THREAD

**Issue by Bravefox** [bmwcarit/joynr#7]
Here is an error from the radio-jee (joynr 18.4). Standalone works correctly. How it can be fixed?

```
Error parsing payload. msgId: 144fc8ce-df9a-4e1f-b3a6-47b0f2714746. from: 144fc8ce-df9a-4e1f-b3a6-47b0f2714746 to: 5ddbb385-3a14-4b6c-9ec2-a6552f0facca. Reason: Unexpected token (END_OBJECT), expected FIELD_NAME: missing property '_typeName' that is to contain type id  (for class java.lang.Object)
 at [Source: {"_typeName":"joynr.Reply","requestReplyId":"d6f1ae05-bb7e-4ede-b9ba-50951d9716cb","response":[{"country":"AUSTRALIA","location":{"_typeName":"joynr.vehicle.GeoPosition","latitude":-37.814107,"longitude":144.96328}}]}; line: 1, column: 215] (through reference chain: joynr.Reply["response"]->Object[][0]). Discarding joynr message.]]
```

Diff:
jee:
```
"response": [
        {
            "country": "AUSTRALIA",
            "location": {
                "_typeName": "joynr.vehicle.GeoPosition",
                "latitude": -37.814107,
                "longitude": 144.96328
            }
        }
    ]
```
standalone:
```
"response": [
        "AUSTRALIA",
        {
            "_typeName": "joynr.vehicle.GeoPosition",
            "latitude": -37.814107,
            "longitude": 144.96328
        }
    ]
```

**Comment by david-katz** [comment]
Looks like you've found a bug in the handling of multi-out params in JEE. We are working on a fix and will provide it in joynr 0.18.5. In the mean time, if you use single-out models, everything should work just fine.

**Comment by david-katz** [comment]
Issue is now fixed.
